emcc.py

```diff
@@ -164,7 +164,7 @@ def __init__(self):
     self.cfi = False
     # Specifies the line ending format to use for all generated text files.
     # Defaults to using the native EOL on each platform (\r\n on Windows, \n on Linux&OSX)
-    self.output_eol = os.linesep
+    self.output_eol = os.linesep.encode()
```

Comment:
on python2, changes like this don't do anything, do they? but i guess it's harmless there?

Reply:
>on python2, changes like this don't do anything, do they?

Right, no effect on Python 2.

os.linesep is byte string on Python 2 and Unicode string on Python 3. It should be bytes here as we are using it to convert binary string from files.

I pushed a new commit to make this more explicit. (`encode()` only when needed)